test/llvmpasses/final-lower-gc.ll

```diff
@@ -67,6 +67,21 @@ top:
   ret {} addrspace(10)* %v
 }
 
+define {} addrspace(10)* @gc_alloc_lowering_var(i64 %size) {
+top:
+; CHECK-LABEL: @gc_alloc_lowering_var
+  %pgcstack = call {}*** @julia.get_pgcstack()
+  %ptls = call {}*** @julia.ptls_states()
+  %ptls_i8 = bitcast {}*** %ptls to i8*
+; CHECK: %0 = add i64 %size, 8
+; CHECK: %v = call noalias nonnull dereferenceable(8) {} addrspace(10)* @ijl_gc_big_alloc(i8* %ptls_i8, i64 %0)
```

Comment:
IIUC. We should not be calling `jl_gc_big_alloc`, but rather `jl_gc_allocobj`.

Reply:
Or rather `jl_gc_alloc_typed(ptls, size, NULL)`. To be precise we should re-use the ptls we obtained here, but `jl_gc_alloc` is not exported.